pkg/dispatchcli/cmd/install.go

```diff
@@ -172,49 +188,83 @@ func NewCmdInstall(out io.Writer, errOut io.Writer) *cobra.Command {
 	}
 
 	cmd.Flags().StringVarP(&installConfigFile, "file", "f", "", "Path to YAML file")
-	cmd.Flags().StringArrayVarP(&installServices, "service", "s", []string{}, "Service to install (defaults to all)")
+	cmd.Flags().StringArrayVarP(&installServices, "service", "s", []string{}, "Service to install (defaults to all). Add 'no-' prefix to service name to disable it.")
 	cmd.Flags().BoolVar(&installDryRun, "dry-run", false, "Do a dry run, but don't install anything")
 	cmd.Flags().BoolVar(&installDebug, "debug", false, "Extra debug output")
 	cmd.Flags().StringVar(&installChartsRepo, "charts-repo", "dispatch", "Helm Chart Repo used")
 	cmd.Flags().StringVar(&installChartsDir, "charts-dir", "dispatch", "File path to local charts (for chart development)")
+	cmd.Flags().StringVar(&installSingleNS, "single-namespace", "", "If specified, all dispatch components will be installed to that namespace")
```

Comment:
Do we need to prefix this option with "single"?  Also, should we install into a single namespace (dispatch) by default?

Reply:
I added the prefix to clearly indicate that the deployment model differs from the default one (multiple namespaces). If we switch to single namespace being default, using just `--namespace` makes more sense. but that's a bigger change (we need to change CI as well), so let's do it in a separate patch.